packages/repack/commands.js

```diff
@@ -50,6 +50,117 @@ function getCommands() {
   return commands;
 }
 
+const startCommandOptions = [
+  {
+    name: '--cert <path>',
+    description: 'Path to custom SSL cert',
+  },
+  {
+    name: '--host <string>',
+    description: '(default: "")',
```

Comment:
can we also for default values use `default` key, same as [here](https://github.com/szymonrybczak/react-native/blob/28357a3eb6fec8699b3d95212d160a9ea2892d4d/packages/community-cli-plugin/src/commands/bundle/index.js#L42)?  `commander` will automatically format these